templates/sso-cd-mysql-persistent.json

```diff
@@ -0,0 +1,838 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass" : "icon-sso",
+            "tags" : "sso,keycloak,jboss,hidden",
+            "version": "7.3.0.CD02",
+            "openshift.io/display-name": "Red Hat Single Sign-On 7.3 CD Tech Preview + MySQL (Persistent with passthrough TLS)",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "description": "An example application based on RH-SSO 7.3 CD Tech Preview image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-7-openshift-image/docs.",
+            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat Single Sign-On 7.3 server based deployment, deployment configuration for MySQL using persistence, and securing RH-SSO communication using passthrough TLS.",
+            "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-single-sign-on/",
+            "template.openshift.io/support-url": "https://access.redhat.com"
+        },
+        "name": "sso-cd-mysql-persistent"
+    },
+    "labels": {
+        "template": "sso-cd-mysql-persistent",
+        "xpaas": "7.3.0.CD02"
```

Comment:
same here, rhsso instead of xpaas.

Reply:
Ok, thanks! I will do sed global replace for them.